### DISCUSSION TITLE
Unable to open terminal on gitbash

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [folke/snacks.nvim#58] [Q&A] [answered]
Based on the latest lazyvim13.1.0

![image](https://github.com/user-attachments/assets/603d59b1-0369-4f76-996a-e365fee63ca3)

Works very well in lazyvim12

![image](https://github.com/user-attachments/assets/3375e7ed-3897-438f-902f-dbee8ea8e096)

**Comment by folke** [comment, marked as answer]
I think this should now be fixed.
Would be great if you could confirm, since I don't do Windows.

**Reply by iamxiaojianzheng** [reply]
solved

**Reply by folke** [reply]
There's another related issue I'm trying to solve.

Can you give me the exact output of `:=vim.o.shell`?

**Reply by iamxiaojianzheng** [reply]
`bash`，Because I configured it manually.

```lua options.lua
vim.o.shell = "bash"
vim.o.shellcmdflag = "-i"
```

**Reply by folke** [reply]
And without configuring it manually?

**Reply by iamxiaojianzheng** [reply]
`C:\Users\xxxx\scoop\apps\git\2.47.0.2\usr\bin\bash.exe`

**Reply by folke** [reply]
I just pushed an update to fix that other issue. Can you verify everything still works for you as well?

**Reply by iamxiaojianzheng** [reply]
work nice

**Comment by iamxiaojianzheng** [comment]
new question.

The lazygit window opens and the line number appears.

![image](https://github.com/user-attachments/assets/949d9540-06c4-46c4-a990-63515f79323a)

**Reply by iamxiaojianzheng** [reply]
If vsplit the window

![image](https://github.com/user-attachments/assets/a418bd2b-d9c6-4a32-9d34-e41a4c8da99b)

If split the window

![image](https://github.com/user-attachments/assets/a1a27172-6db0-4e57-9a5a-109f324f393a)

**Reply by folke** [reply]
I'm not sure how you start that split, but you can do something like:

```lua
lua Snacks.lazygit({win = {position = "right", width=0.5}})
```

![image](https://github.com/user-attachments/assets/9e4102a9-46fd-414a-ac3c-7261ac1f7edb)

**Reply by iamxiaojianzheng** [reply]
use `<leader>wv` or `<leader>ws` to split window.

It is normally expected that line numbers should not be displayed

**Reply by iamxiaojianzheng** [reply]
I customized some configurations

```lua
{
  "folke/snacks.nvim",
  priority = 1000,
  lazy = false,
  -- cond = false,
  ---@type snacks.Config
  opts = {
    lazygit = {
      win = {
        width = 0,
        height = 0,
      },
    },
    terminal = {
      win = {
        position = "right",
      },
    },
  },
}
```

**Reply by iamxiaojianzheng** [reply]
I just found a solution, but it seems to be a bug.

terminal's position configuration seems to affect lazygit.

```lua
{
  "folke/snacks.nvim",
  priority = 1000,
  lazy = false,
  ---@type snacks.Config
  opts = {
    lazygit = {
      win = {
        width = 0,
        height = 0,
        position = "float", -- Adding this line can also be solved
      },
    },
    terminal = {
     -- Or commenting out the win configuration can also be resolved.
      win = {
        position = "right",
      },
    },
  },
}
```
